src/task11/task11

```diff
@@ -0,0 +1,35 @@
+var ancestry = JSON.parse(ANCESTRY_FILE);
+
+const average = array => array.reduce((a, b) => a + b) / array.length;
+
+var byCentury = {};
+ancestry.forEach(person => {
```

Comment:
```
ancestry.forEach(person => [person.name] = person);
```

Reply:
regarding formatting
it marks ; red if I write `ancestry.forEach(person => [person.name] = **person;**);`
why?

Reply:
because the compiler doesn't think it's correct
not correct for ES6+
could be correct for ES5:
```
... => {
    return smth;
}
```